src/hooks/api/getters/useFeatures/useFeatures.ts

```diff
@@ -0,0 +1,40 @@
+import useSWR, { mutate, SWRConfiguration } from 'swr';
+import { useState, useEffect } from 'react';
+import { formatApiPath } from '../../../../utils/format-path';
+import handleErrorResponses from '../httpErrorResponseHandler';
+
+const useFeatures = (options: SWRConfiguration = {}) => {
+    const fetcher = async () => {
+        const path = formatApiPath('api/admin/features/');
```

Comment:
Should move this out of the fetcher. We don't need to recreate the path on every fetch.

Reply:
True! I've moved the path out in my branch, so I can fix when doing the rebase:

https://github.com/Unleash/unleash-frontend/pull/663/files#diff-b4d115db9cc2bc0a0e4c3d8a5f74cdac2c83fa528ca19dde4bbdb1bffd328aaeR7